keyboards/stoutgat/info.json

```
@@ -0,0 +1,89 @@
+{
+    "keyboard_name": "stoutgat",
+    "url": "https://github.com/vattern/stoutgat",
+    "maintainer": "Thys de Wet",
```

This is meant to be your GitHub username.

```suggestion
    "maintainer": "vattern",
```